message/tracing.go

```diff
@@ -0,0 +1,24 @@
+package message
+
+import (
+	"context"
+
+	servicebus "github.com/Azure/azure-service-bus-go"
+	"github.com/devigned/tab"
+)
+
+func startSpanFromMessageAndContext(ctx context.Context, operationName string, message *servicebus.Message) (context.Context, tab.Spanner) {
+	ctx, span := tab.StartSpan(ctx, operationName)
+	if message != nil {
```

Comment:
this should never happen. Log an erro when it does?

Reply:
Log an error when msg is nil.